database/GET.go

```diff
@@ -0,0 +1,399 @@
+package database

```

Comment:
1. Use `context.Context` for database calls: Pass a context through the function calls and use it with the `QueryRowContext` and `QueryContext` methods to allow for proper cancellation and timeouts.

2. Replace `any` with `interface{}`: The `[]any` type is not valid in Go. Replace it with `[]interface{}`.

3. Error handling: Make sure to check the error returned by `rows.Scan()` even inside a loop. This will help in debugging issues if any.

4. Use constants for case types: In the `GetTopCountriesByCaseTypeForUser` function, consider using constants for case types like "confirmed", "deaths", and "recovered" instead of string literals.

5. Remove `fmt.Println()` calls in production code: Remove debugging calls like `fmt.Println()` before deploying the code to production.

6. Avoid SQL injections: In the `buildTopCountriesByCaseTypeForUserQuery` function, directly concatenating the `caseType` string into the SQL query string might lead to SQL injection. Validate the input